packages/manager/src/utilities/analytics.ts

```diff
@@ -8,19 +8,13 @@ declare global {
 }
 
 type DTMSatellite = {
-  track: (
-    eventName: string,
-    eventPayload: AnalyticsEvent | PageViewEvent
-  ) => void;
+  track: (eventName: string, eventPayload: Record<string, unknown>) => void;
 };
 
-interface PageViewEvent {
-  url: string;
-}
-
 interface AnalyticsEvent {
   action: string;
   category: string;
+  details?: Record<string, unknown>;
```

Comment:
```suggestion
  data?: Record<string, unknown>;
```
I think Richa went with `data` here, rather than `details`, so that's a change we would need to make to get this to the backend. 

I was also thinking that we create an interface for the `data` (details) object:
```
export interface CustomAnalyticsData {
  isLinodePoweredOff?: boolean;
  value?: number; //We *could* move our existing "value" field here
  //We could add other things in future
}
```
so that if we wanted to add something new to our `data`, we could, and we're making sure we're typing it correctly. Thoughts?